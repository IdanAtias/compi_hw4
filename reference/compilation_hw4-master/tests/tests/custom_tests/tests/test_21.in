matrix m(2,3) = [1, 2, 3; 4, 5, 6];
// Testing wrong dimensions
m ^ (2);

