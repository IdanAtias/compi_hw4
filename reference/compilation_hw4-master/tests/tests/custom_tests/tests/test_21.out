m=
1,2,3
4,5,6
ArgumentSizeMismatch with operation ^. left argument size 3 and right argument size 2
